recipes/r-phewas/meta.yaml

```diff
@@ -24,7 +24,7 @@ build:
 requirements:
   build:
     - gcc_linux-64
-    - gofrtran_linux-64
+    - gfortran_linux-64
```

Comment:
it would be great if you change that to:

```
- {{ compiler('c') }}
- {{ compiler('fortran') }}
```

:)

Reply:
Hi @marcelotrevisani I tried with jinja functions but did not work. will try again.